src/helpers/discord_utils.ts

```diff
@@ -218,55 +219,60 @@ export async function sendOptionsMessage(messageContext: MessageContext, guildPr
     optionStrings[GameOption.GOAL] = `${guildPreference.getGoal()}`;
     optionStrings[GameOption.TIMER] = `${guildPreference.getGuessTimeout()}`;
     optionStrings[GameOption.SHUFFLE_TYPE] = `${guildPreference.getShuffleType()}`;
     optionStrings[GameOption.SUBUNIT_PREFERENCE] = `${guildPreference.getSubunitPreference() === SubunitsPreference.INCLUDE ? "including" : "excluding"} subunits`;
     const ostPreferenceDisplayStrings = {
-        [OstPreference.INCLUDE]: "Including",
-        [OstPreference.EXCLUDE]: "Excluding",
-        [OstPreference.EXCLUSIVE]: "Exclusively including",
+        [OstPreference.INCLUDE]: "include",
+        [OstPreference.EXCLUDE]: "exclude",
+        [OstPreference.EXCLUSIVE]: "exclusive include",
     };
     optionStrings[GameOption.OST_PREFERENCE] = `${ostPreferenceDisplayStrings[guildPreference.getOstPreference()]}`;
-    optionStrings[GameOption.RELEASE_TYPE] = `${guildPreference.getReleaseType() === ReleaseType.OFFICIAL ? "only official song releases" : "all songs including dance practices, acoustic versions, remixes, etc"}`;
-    optionStrings[GameOption.MULTIGUESS] = guildPreference.getMultiGuessType() === MultiGuessType.ON ? "Multiple players are able to guess correctly." : "";
+    optionStrings[GameOption.RELEASE_TYPE] = `${guildPreference.getReleaseType() === ReleaseType.OFFICIAL ? "official" : "all including dance practices, acoustic versions, remixes, etc"}`;
+    optionStrings[GameOption.MULTIGUESS] = guildPreference.getMultiGuessType() === MultiGuessType.ON ? "on" : "off";
 
     for (const gameOption of Object.keys(optionStrings)) {
         const gameOptionString = optionStrings[gameOption];
         if (!gameOptionString) continue;
         optionStrings[gameOption] = (updatedOption && updatedOption.option) === gameOption ? bold(gameOptionString) : codeLine(gameOptionString);
     }
 
-    const goalMessage = `First one to ${optionStrings[GameOption.GOAL]} points wins.`;
-    const guessTimeoutMessage = ` in less than ${optionStrings[GameOption.TIMER]} seconds`;
-    const shuffleMessage = `Songs will be shuffled in ${optionStrings[GameOption.SHUFFLE_TYPE]} order. `;
+    const goalMessage = optionStrings[GameOption.GOAL];
+    const guessTimeoutMessage = optionStrings[GameOption.TIMER];
+    const shuffleMessage = optionStrings[GameOption.SHUFFLE_TYPE];
 
     if (updatedOption && updatedOption.reset) {
-        footerText = `Looking for information on how to use this command? Check out '${process.env.BOT_PREFIX}help [command]' to learn more`;
+        footerText = `Looking for information on how to use this command? Check out '${PREFIX}help [command]' to learn more`;
     }
 
     await sendInfoMessage(messageContext,
         {
             title: updatedOption === null ? "Options" : `${updatedOption.option} ${updatedOption.reset ? "reset" : "updated"}`,
             description:
```

Comment:
See if you can abstract all the options below into something that creates an Discord embed of options. Look into [Embed fields](https://discord.com/developers/docs/resources/channel#embed-object-embed-field-structure). Then refer to `scoreboard.ts`'s `getScoreboardEmbedFields()` as an example of how to create a field for every option (`name` would be the option, `value` would be what the option is set to, and for now all of them should be inline). Instead of passing this array of fields to `sendInfoMessage`'s `EmbedPayload`'s `description` like it is now, pass it to `sendInfoMessage`'s `EmbedPayload`'s `fields` property.